consensus/src/msg_handler.rs

```diff
@@ -78,6 +80,16 @@ pub trait MsgHandler<T: Debug + MessageTrait> {
         committee: &Committee,
     ) -> Result<HandleMsgOutput, ConsensusError>;
 
+    /// collect allows each Phase to process a verified message from a former
+    /// iteration
+    async fn collect_from_past(
+        &mut self,
+        msg: T,
+        ru: &RoundUpdate,
```

Comment:
seems to me that ru is never used in any implementation. could it be safely removed?